### ISSUE TITLE
EcAdapterFactory doesn't detect secp256k1-php

### ISSUE THREAD

**Issue by andkom** [Bit-Wasp/bitcoin-php#777]
secp256k1 version doesn't match

```
if (extension_loaded('secp256k1') && Semver::satisfies(phpversion('secp256k1'), "~0.1.0"))

extension_loaded('secp256k1') // true
phpversion('secp256k1') // 2.0
```

**Comment by afk11** [comment]
Damn :/ thanks, will check that out

**Comment by afk11** [comment]
The fix for this was tagged in v1.0.1 - thanks again for the report!
